assets/scss/moj-blocks-general.scss

```diff
@@ -1,3 +1,11 @@
+.hale-page {
+  // Deals with the Hero and the highlight/shaded blocks (e.g. CTA, highlighted list)
+  // The shading for these have (need) widths of over 100vw which leads to undesirable horizontal scrolling
+  // No content is outside of the page area, just highlighting.
+  // This prevents this from being an issue by preventing horizontal scroll
+  overflow-x: hidden;
+}
+
```

Comment:
Is this just something that will effect those two blocks or do you think it is worth applying to the wider site?

Reply:
This will apply to the wider site.  If there are other blocks that have content (as opposed to just styling) disappearing off the edge, that block will either need to be able to horizontal scroll itself or another fix will be needed.  I couldn't find any such issue at the moment.